app/configurator/components/ui-helpers.spec.ts

```diff
@@ -75,6 +83,11 @@ describe("useDimensionFormatters", () => {
     const { formatters } = setup();
     expect(formatters["iri-number"]("2.33333")).toEqual("2,33");
   });
+
+  it("should work with currencies", () => {
+    const { formatters } = setup();
+    expect(formatters["iri-currency"]("20002.3333")).toEqual("20'002,3");
+  });
```

Comment:
Just wondering, if you're using locale in this formatter (currencyFormatter), can the test fail in some scenarios if run with a locale that use different formatting? Or will it be the same in all cases?

Reply:
It could fail if run with a different locale, but for now we do not test for this.